features/org.wso2.carbon.core.common.feature/pom.xml

```diff
@@ -116,6 +116,9 @@
                                 <bundleDef>org.wso2.orbit.org.apache.tomcat:jdbc-pool:${jdbc-pool.version}</bundleDef>
                                 <bundleDef>org.eclipse.jdt.core.compiler:ecj:${version.eclipse.ecj}</bundleDef>
                                 <bundleDef>com.google.code.gson:gson:${version.com.google.code.gson}</bundleDef>
+                                <!--commons-text and commons-lang3 required for axiom-->
+                                <bundleDef>org.wso2.orbit.commons-text:commons-text:${commons-text.orbit.version}</bundleDef>
```

Comment:
Shall we just use the maven dependency , it is already osgi ready , isn't it ?

Reply:
Yes it is. But all the other bundles in this feature are of wso2 versions. Since we already have orbit bundles for these, I'm just using them, so that I don't want to break the convention.